Payment UseCase/Payment.java

```diff
@@ -2,6 +2,7 @@
 //The class and methods of Presentation layer
 //Presentation layer (interation to user) To choose the method "Credit Card , Debit Card ,and Benefit Pay " to Payment
 
+
```

Comment:
//add Enum to define the payment methods((

Reply:
Ilike to put Enum method that ensuring only predefined method of payment can be used